docs/spec/api.md

```diff
@@ -1108,7 +1133,9 @@ A list of methods and URIs are covered in the table below:
 |Method|Path|Entity|Description|
 |------|----|------|-----------|
 | GET | `/v2/` | Base | Check that the endpoint implements Docker Registry API V2. |
-| GET | `/v2/<name>/tags/list` | Tags | Fetch the tags under the repository identified by `name`. |
+| GET | `/v2/<name>/tags/_list` | Tags | Fetch the tags under the repository identified by `name`. |
+| GET | `/v2/<name>/tags/<reference>` | Tags | Redirect to tags list. |
```

Comment:
That part of template is generated from `/registry/api/v2/descriptors.go` Over there it's just one route for GET and DELETE with uses `<reference>` to get correct tag. It works same way for DELETE and GET and if we were developing from scratches we would not implement GET at all. But we have to be backward compatible, hence the only `<reference>` supported in GET is `list` and that's just a special case. I see no reason to define a special route and handler for it.